### PR TITLE
[CUBRIDMAN-75] A new index on _db_domain and fix some errors about indexes on catalog classes

### PR DESCRIPTION
**Pull request by wrlawodms**
http://jira.cubrid.org/browse/CUBRIDMAN-75

en: http://192.168.2.101:9001/sql/catalog.html#db-domain
ko: http://192.168.2.101:9000/sql/catalog.html#db-domain

To access the new man, you have to connect to CUBRID OFFICE VPN.

Some fixes not related to _db_domain are fixed. Please see the Jira issue.

### CODE REVIEW

**Review by mhoh3963 — APPROVED** [review]

**Review by charischau — APPROVED** [review]

**Review by joohok — APPROVED** [review]

**Review by eido5 — APPROVED** [review]

File: ko/sql/catalog.rst [hunk 1]
```diff
@@ -259,7 +259,7 @@ _db_attribute
 _db_domain
 ----------
 
-도메인에 대한 정보이며 object_of에 대한 인덱스가 생성되어 있다.
+도메인에 대한 정보이며 object_of에 대한 인덱스와 data_type에 대한 인덱스가 생성되어 있다.
```
Comment (hornetmj) [inline ko/sql/catalog.rst]:
```suggestion
도메인에 대한 정보이며, object_of와 data_type에 대한 인덱스가 생성되어 있다.
```

**Review by hornetmj — APPROVED** [review]

File: ko/sql/catalog.rst (hunk 1 above)
Comment (wrlawodms) [reply ko/sql/catalog.rst]: I want to distinguish two cases: one index for several columns (other catalog cases) and two indexes for each column (_db_domain case). What you suggested is other's way.

Comment (hornetmj) [reply ko/sql/catalog.rst]: You mean that my suggestion looks like a multiple-column index on the object_of, data_type. My opinion pointed to the readability. I thought using '대한 인덱스' repeatably prevent the readability. So, I suggest "도메인에 대한 정보이며, object_of와 data_type 컬럼에 각각(개별적인) 인덱스가 생성되어 있다.". But I fully understand what you mean. So, you can choose one of the conditions (apply my suggestion or leave it as before.)

Comment (wrlawodms) [reply ko/sql/catalog.rst]: I meant the description of others with only one index for two columns looks like yours. I agree with the readability problem, though. I'll revise it as you said.

Comment (wrlawodms) [reply ko/sql/catalog.rst]:
```suggestion
도메인에 대한 정보이며, object_of와 data_type 컬럼에 각각(개별적인) 인덱스가 생성되어 있다.
```

**Pull request merged**